ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/ClaimAssert.java

```diff
@@ -262,15 +262,15 @@ public ClaimAssert isEqualTo(CCDCase ccdCase) {
         actual.getReviewOrder()
             .ifPresent(reviewOrder -> assertThat(reviewOrder).isEqualTo(ccdCase.getReviewOrder()));
 
-        if (ccdCase.getChannelType() != null && !actual.getChannelType().isPresent()) {
-            failWithMessage("Expected CCDCase.channelType to be not present but was <%s>",
-                actual.getChannelType());
+        if (ccdCase.getChannel() != null && !actual.getChannel().isPresent()) {
+            failWithMessage("Expected CCDCase.channel to be not present but was <%s>",
+                actual.getChannel());
```

Comment:
The test and the message are inverted

Reply:
Good catch. Fixed.

Reply:
This and other mistakes like it will be fixed in tech debt https://tools.hmcts.net/jira/browse/ROC-6549